example-client/README.md

```diff
@@ -3,11 +3,219 @@ Example CoAP client
 
 This is an example coap client application, that can be use with any LwM2M Server.
 
```

Comment:
- coap -> CoAP
- ..can be used ( -> or rather active voice, ... that you can use with...)
- Server -> server (no need to capitalize)